en/micro-integrator/docs/setup/installation/run_in_docker.md

```diff
@@ -40,6 +40,76 @@ Two docker images are available for the Micro Integrator:
     ```bash
     wso2/micro-integrator:1.0.0
     ```
+    
+## Build and run on Docker using Integration Studio
+
+Let’s create a simple proxy service using Integration studio and deploy it in a docker container. 
+
+For this we can use following synapse config
+
+```
+<?xml version="1.0" encoding="UTF-8"?>
+<proxy name="HelloWorld" startOnLoad="true" transports="http https" xmlns="http://ws.apache.org/ns/synapse">
+    <target>
+        <inSequence>
+            <payloadFactory media-type="json">
+                <format>{"Hello":"World"}</format>
+                <args/>
+            </payloadFactory>
+            <respond/>
+        </inSequence>
+        <outSequence/>
+        <faultSequence/>
+    </target>
+</proxy>
+```
+
+So now we can create the docker project with config following the steps 
```

Comment:
Have an opening sentence as follows:

"To create a Docker image with the Synapse configuration you added, follow the procedure below:"

Then have the instructions as numbered steps.